Preferences UI tweaks

ronso0 opened the pull request:
Just a few commits off my stash list. See commits for details.

7b83ac784bf465c3984af7b17bc1eb590e5cc1db contains a new tr string, let me know if we should rather merge that to main.

ronso0 committed:
Preferences: const shared warning icon string (html)

ronso0 committed:
Pref Library: mark missing/invalid directories with icon, add tooltip

ronso0 committed:
Pref Controller: reduce minimum width of root page

Minimum width is now either width of the links at the bottom or total
width of buttons (depends on translations).

Holzhaus commented:
> 7b83ac7 contains a new tr string, let me know if we should rather merge that to main.

Just wondering, when was the last time we pushed translatable strings to transifex? We probably should do that soon to avoid ending up with a bunch of untranslated stuff when we release 2. 4.

daschuer commented:
I have just updated the translations.

daschuer commented:
If the string change is not fixing wrong information, it is better to move it to 2.5. New strings can be added at anytime.

ronso0 commented:
FYI it's no important UI string, just a tooltip for music directories in the Library preferences that are currently not available / invalid.
I don't mind moving the commit.

daschuer commented:
I have no special demands here, both works for me.

ronso0 committed:
Pref Interface: group into Skin and Misc, remove unneeded spacer

ronso0 committed:
Pref Waveform: keep dialog as narrow as possible

by adding QGroupBox 'OpenGL status' (remove the respective label) so the
lengthy OpenGL version string can use the entire width

ronso0 committed:
Pref Library: allow breaking some long labels

ronso0 committed:
Pref Sound: remove bottom spacer

ronso0 committed:
Pref Broadcast: disable Ogg checkbox if not relevant

ronso0 committed:
Pref Broadcast: move Metadata options below Stream Info options

ronso0 committed:
Pref Recording: make file format options independent from encoder grids

ronso0 committed:
Pref Decks: remove meaning less sensitivity spinbox

daschuer reviewed (approved):
Works good and looks good. The string is helpful even if it sticks untranslated. 
Thank you.

ronso0 commented on src/preferences/dialog/dlgprefinterface.cpp:
```diff
@@ -125,12 +125,7 @@ DlgPrefInterface::DlgPrefInterface(
-                tr("The minimum size of the selected skin is bigger than your "
-                   "screen resolution.");
-        warningLabel->setText(sizeWarningString);
+        warningLabel->setText(kWarningIconHtmlString);
```
I'll prepare a fixup asap

ronso0 replied:
Uff, damn, hopefully the translations have not been dumped, yet.
@daschuer (or whoever is going to push tr) please wait for the fixup!
